package.json

```diff
@@ -2,6 +2,8 @@
   "name": "samples-node",
   "private": true,
   "scripts": {
+    "pretest": "cd activities-examples && npm install",
+    "test": "cd activities-examples && npm test",
```

Comment:
Maybe use [pushd and popd](https://fishshell.com/docs/current/cmds/pushd.html) instead so this doesn't change the user's cwd? Also, would be great if we didn't have to explicitly list all examples here. Maybe we write a test script instead?

Reply:
Will try switching to lerna

Reply:
pushd and popd don't matter here, script runs in a different shell